CHANGELOG.rst

```diff
@@ -43,6 +47,7 @@ Miscellaneous:
 * Pinned ``hypothesis<6.0.4`` to avoid
   `python/cpython#102126 <https://github.com/python/cpython/issues/102126>`_
   in Python 3.10.10 and 3.11.2
+* Updated JSON Schema Test Suite to commit fd80307
```

Comment:
This doesn't need to go in the change log. Submodules should all be up-to-date at package release time.

Reply:
@marksparkza I am happy to hear that!  I'm fairly certain not everyone is as considerate about submodule synchronization 😐